pyomo/solvers/plugins/solvers/XPRESS.py

```diff
@@ -303,7 +306,8 @@ def process_logfile(self):
                 results.solver.termination_message = ' '.join(tokens)
 
         try:
-            results.solver.termination_message = yaml_fix(results.solver.termination_message)
+            if isinstance(results.solver.termination_message, basestring):
```

Comment:
This could just be `six.string_types`, right?

Reply:
It, in fact, could be. There's a lot of mix-and-match through Pyomo about whether to use `str`, `basestring`, or `string_types`. We may want to decide and standardize.

Reply:
I have opened #1823 and assigned it to myself.